CHANGELOG.md

```diff
@@ -1,7 +1,20 @@
 Change Log
 ==========
 
-Version 3.0.7 *2019-01-31*
+## Version 3.1.0 *(2019-07-30)*
+
+### Breaking‼️:
+
+* [Remove XingApi's resource function](https://github.com/xing/xing-android-sdk/pull/245)
+  * The reasons for this change are:
+    * `XingApi` is a dependency of any `Resource`, therefore it doesn't make sense that the former provides or holds a
+    Resource in any way.
+    * `XingApi` also used to hold a cache of resources, this could potentially cause problems if an implementation would
+    contain any mutable state inside, as it would be propagated to further calls to the `resource` function.
+    * Finally, this function used to sometimes rely on reflection, which is not mentioned anywhere and leads to runtime
+    crashes when the target implementation Proguard's strips out the resources.
```

Comment:
I suggest to add a snippet to what need to be done in order to migrate to a new version. Or just a mention that `GETTING_STARTED_ANDROID.md` has updated version of the resource creation.